administrator-guides/integrations/github/README.md

```diff
@@ -454,31 +539,32 @@ const githubEvents = {
             });
         } else{
             attachment.fields.push({
+                title: changeset,
                 value: output,
                 short: is_short
             });
         }
       } else {
         output = "[" + shortID + "](" + commit.url + ")"
+        added =  commit.added 
         attachment.fields.push({
             title: changeset,
             value: output,
             short: is_short
-        });
+        }); 
       }
     }
 
     const text = ':ballot_box_with_check: Pushed to ' + "["+request.content.ref.split('/').pop()+"]";
 
     return {
       content: {
-        icon_url: user.avatar_url,
         alias: user.login,
         text: text,
         attachments: [attachment]
       }
   };
-  },  // End GitHub Push
+  },  // End Github Push
```

Comment:
Please keep GitHub with a capital H, as they prefer like that... see the footer of github.com :)

Reply:
Ok Done